test/format/util.c

```diff
@@ -428,30 +177,20 @@ path_setup(const char *home)
 uint32_t
 rng_slow(WT_RAND_STATE *rnd)
 {
-    u_long ulv;
     uint32_t v;
-    char *endptr, buf[64];
 
     /*
      * We can reproduce a single-threaded run based on the random numbers used in the initial run,
      * plus the configuration files.
      */
     if (g.replay) {
-        if (fgets(buf, sizeof(buf), g.randfp) == NULL) {
-            if (feof(g.randfp)) {
-                fprintf(stderr,
-                  "\n"
-                  "end of random number log reached\n");
-                exit(EXIT_SUCCESS);
-            }
-            testutil_die(errno, "random number log");
+        if (fp_readv(g.randfp, g.home_rand, true, &v)) {
+            fprintf(stderr,
```

Comment:
If you pull this all into one line (it looks like it will fit), will clang-format leave it unchanged?

Reply:
Clang-format insists.